CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+##[0.15.5] - released on 2018-03-05
```

Comment:
formatting looks wonky, i think you need a space between the ## and [

Reply:
whoops!